### ISSUE TITLE
[Bug]: Accessibility addon: Violation content is not correct

### ISSUE THREAD

**Issue by MichaelArestad** [storybookjs/storybook#31414]
### Describe the bug

1. ~The violation description should match Chromatic's descriptions. They are not custom so it should be a matter of selecting the corresponding field.~
2. The violation link should say “Learn how to resolve this violation”
3. The violation listed in a popover should not be a menu item and should include a link to go to axe’s documentation. I will share screenshots of the specification below.
4. ~The violation listed in a popover should match the title, description, and link shown in the panel.~

<img width="1069" alt="Image" src="https://github.com/user-attachments/assets/5fa05a17-6b9a-4670-9250-628db4a8e263" />

### Reproduction link

https://635781f3500dd2c49e189caf-xifgkzulyi.chromatic.com/?path=/story/button-component--variants

### Reproduction steps

1. Go to the above URL
2. Open the Accessibility panel
3. Open "Color contrast." The description says "The color contrast between text and its background meets WCAG AA contrast ratio." The description **should** say "Elements must meet minimum color contrast ratio thresholds"
4. Also check the issues listed above: Incorrect link text and popover menu item behavior

### System

```bash
No, ty.
```
